app/api/gql/resolver/mutation.go

```diff
@@ -50,6 +51,9 @@ func (m Mutation) CreateTask(
 	if len(activeTeams) == 0 {
 		return Task{}, fmt.Errorf("user %v does not have an active team", userID)
 	}
+
+	task.Status = entity.UPCOMING
+	task.OwnerUserId = &userID
```

Comment:
```suggestion
	task.OwnerUserId = &userID
```

I think we need take this value from `taskInput`